css/style.css

```diff
@@ -0,0 +1,10 @@
+#img-div {
+  display: block;
+  margin: auto;
+}
+
+img {
+  display: block;
+  margin: auto;
+  max-width: 100%;
+}
```

Comment:
For proper work in CSS, it would be better to pay attention to start with your main important aspects from HTML to stylize how it would be the body, the main etc.

Reply:
* I suggest you make your comments clear and easy to understand so that student know what is expected of them to change 👍

Reply:
- I would suggest you start your work in the CSS with the main tags as <body>, <main>, <header> etc.

Reply:
Great! But I will suggest starting with adding display styles to the divs or the containers tags first in order to have a visual structure and only after that style the inner tags.